CONTRIBUTING.md

```diff
@@ -289,6 +289,7 @@ python3 -m pip install -r tools/install_deps/pytest.txt
 
 Compile the custom ops
 ```
+export TF_NEED_CUDA=1 # If GPU is to be used
```

Comment:
I think that we need to set this only in the GPU image

Reply:
Yes, but also if it's built from source locally.